moveit_setup_assistant/src/widgets/start_screen_widget.h

```diff
@@ -55,8 +55,7 @@ namespace moveit_setup_assistant
 {
 // Class Prototypes
 class SelectModeWidget;
-class LoadPathWidget;
-// class LoadURDFWidget;
+class LoadPathArgsWidget;
```

Comment:
i don't like how you renamed this class, can we keep old name and not have two different classes?

Reply:
See above. A better name is welcome ;-)